### ISSUE TITLE
 Error: Cannot find module '@napi-rs/snappy-linux-x64-gnu' - on M1 Mac

### ISSUE THREAD

**Issue by askprateek** [Brooooooklyn/snappy#84]
<img width="529" alt="image" src="https://user-images.githubusercontent.com/9272830/186384993-31e98b30-b2d9-4b52-b85c-664da1449eb0.png">

I'm getting this error while running the app via docker-compose

**Snappy:** 7.1.2
**Laptop:** M1 Macbook Pro

**Comment by Brooooooklyn** [comment]
@askprateek have you run the installation in the Docker container?

**Comment by Brooooooklyn** [comment]
@askprateek are you using npm? there is a bug in `npm` related: https://github.com/npm/cli/issues/4828.

**Comment by askprateek** [comment]
We were able to solve this.
We forcefully installed the required package to make it work.

Our understanding is 
-  You run npm install on M1 mac. It installs darwin binaries.
- You run docker build: it creates an image on architecture: linux/x64. We got this info using the docker inspect command.
- This is resulting in a mismatch. 

To solve this we ran `npm i @napi-rs/snappy-linux-arm64-gnu --save-dev -f` to forcefully install the package on my Mac M1 machine to temp fix the dependency issue.
